src/OfficeService.Validation/Users/RemoveOfficeUsersValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.OfficeService.Data.Interfaces;
+using LT.DigitalOffice.OfficeService.Models.Dto.Requests.Users;
+using LT.DigitalOffice.OfficeService.Validation.Users.Interfaces;
+
+namespace LT.DigitalOffice.OfficeService.Validation.Users
+{
+  public class RemoveOfficeUsersValidator : AbstractValidator<RemoveOfficeUsers>, IRemoveOfficeUsersValidator
+  {
+    public RemoveOfficeUsersValidator(
```

Comment:
I think there is no need for this validator, if you do not find users in this office in the repository, then you will not delete them from there